__tests__/app.test.js

```diff
@@ -38,19 +35,57 @@ describe("app", () => {
           });
       });
     });
-    describe("/api/", () => (
-        test("GET 200: should return with an object describing all the other available endpoints on /api/", () => {
-            return fs.readFile("/home/alex/northcoders/backend/project/be-nc-news/endpoints.json",
-            "utf-8")
-            .then((endpoints) => {
-                return request(app)
-                .get("/api/")
-                .expect(200)
-                .then(({body}) => {
-                 expect(body.endpoints).toEqual(JSON.parse(endpoints))   
-                })
-            })
+    describe("/api/", () =>
+      test("GET 200: should return with an object describing all the other available endpoints on /api/", () => {
+        return fs
+          .readFile(
+            `${__dirname}/../endpoints.json`,
+            "utf-8"
+          )
+          .then((endpoints) => {
+            return request(app)
+              .get("/api/")
+              .expect(200)
+              .then(({ body }) => {
+                expect(body.endpoints).toEqual(JSON.parse(endpoints));
+              });
+          });
+      }));
+    describe("/api/articles/:article_id", () => {
+      test("GET 200: should return with an article object with all the correct properties", () => {
+        return request(app)
+          .get("/api/articles/1")
+          .expect(200)
+          .then(({ body }) => {
+            expect(body.article).toEqual({
+             article_id: 1,
+              title: "Living in the shadow of a great man",
+              topic: "mitch",
+              author: "butter_bridge",
+              body: "I find this existence challenging",
+              created_at: "2020-07-09T20:11:00.000Z",
+              votes: 100,
+              article_img_url:
+                "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700",
```

Comment:
This is great at the moment, but later on if the endpoint adds more properties (spoiler - it will!), can we use a more flexible matcher rather than toEqual, so this test would still pass?